core/actions/notebook.ts

```diff
@@ -0,0 +1,35 @@
+import { verifyObjectMatchesProto } from "df/common/protos";
+import { IActionBuilder } from "df/core/actions";
+import { Session } from "df/core/session";
+import { dataform } from "df/protos/ts";
+
+/**
+ * @hidden
```

Comment:
This? https://typedoc.org/tags/hidden

So your intention is to hide the class from docs until you're ready, is that right?

Reply:
We don't expose these action classes anyways, because users don't access them directly https://github.com/dataform-co/dataform/blob/a4a0a6086a7baa0c6a694dada3895608766a8c9e/core/table.ts#L381